src/features/Post/components/PostItem/index.jsx

```diff
@@ -6,25 +6,48 @@ import {
 	Comment,
 	MoreHoriz,
 } from "@mui/icons-material";
-import React, { useState } from "react";
+import React, { useEffect, useState } from "react";
 import { getPostModalState } from "features";
 import { useDispatch, useSelector } from "react-redux";
 import {
 	EDIT_MODAL_VISIBILITY,
 	SET_POST_TO_BE_EDITED,
 	getAuthState,
 	deletePost,
+	likePost,
+	unlikePost,
 } from "features";
-import { useToast } from "hooks";
+import { useLoadPostServices, useToast } from "hooks";
 
 const PostItem = ({ post }) => {
 	const [showMoreOptions, setShowMoreOptions] = useState(false);
 	const {
 		authUser: { username: authUsername },
 		authToken,
 	} = useSelector(getAuthState);
+	const { isLoadingService, handleChangeLoadingServiceState } =
+		useLoadPostServices();
+	const { loadingLikeService } = isLoadingService;
 
-	const { content, username, _id } = post;
+	const {
+		content,
+		username,
+		_id,
+		likes: { likeCount, likedBy },
+	} = post;
+
+	const getIsPostLikedByAuthUser = () =>
+		likedBy.find((likedUsers) => likedUsers.username === authUsername)
+			? true
+			: false;
```

Comment:
Instead of using ternary, just leave the expression as it is. As it'll also do the same job of returning a boolean value.

Reply:
find() returns the first found element, if it exists in the array or undefined, hence the use of ternary and returning a boolean.